app/admin/imports/route.js

```diff
@@ -0,0 +1,37 @@
+import Ember from 'ember';
+import moment from 'npm:moment';
+
+var DATE_FORMAT = 'YYYY-MM-DD';
+
+export default Ember.Route.extend({
+  queryParams: {
+    at: {
+      refreshModel: true
+    }
+  },
+  model({at}) {
+    var currentDate = moment().format(DATE_FORMAT);
+    return this.store.ajax({
+      url: '/api/v1.0/reports/raw',
+      query: {
+        at: at || currentDate
+      }
+    }).then(rawData => ({
+        rawData
```

Comment:
I don't think this `then` is necessary if you change `setupController` to this:

setupController(controller, rawData)